src/api/missionsAPI.js

```diff
@@ -0,0 +1,20 @@
+const url = 'https://api.spacexdata.com/v3/missions';
```

Comment:
Hi @Nuri1977,
Status: Approved 
Excellent job on this project. Keep it up.
Data fetched from API and rendered used hook UseSelector.
Your project is complete! There is nothing else to say other than... it's time to merge it !
Congratulations! 

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.